.github/container/build-jax.sh

```diff
@@ -133,10 +133,6 @@ while [ : ]; do
             XLA_ARM64_PATCH_LIST=$2
             shift 2
             ;;
-        --other-xla-patches)
```

Comment:
Keeping that attribute could be useful in the future.
We can always add it back later, but having it for dev could be useful.